src/utils/event_loop.c

```diff
@@ -268,6 +268,8 @@ event_loop_initialize(struct event_loop *loop)
     int ret = event_loop_epoll_initialize(loop);
     if (ret < 0)
         return ret;
+    
+    loop->exit_code = 0;
```

Comment:
Should this be done in `event_loop_epoll_initialize` instead?

Reply:
is the exit_code related to epoll?

Reply:
Oops sorry, no, you're right, this is the right place.